src/Microsoft.NetCore.Analyzers/Core/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1062,4 +1062,13 @@
   <data name="DoNotCreateTasksWithoutPassingATaskSchedulerMessage" xml:space="preserve">
     <value>Do not create tasks without passing a TaskScheduler</value>
   </data>
+  <data name="UseAutoValidateAntiforgeryToken" xml:space="preserve">
+    <value>Use antiforgery tokens in ASP.NET Core MVC controllers</value>
+  </data>
+  <data name="UseAutoValidateAntiforgeryTokenDescription" xml:space="preserve">
+    <value>Handling a POST, PUT and DELETE request without verifying that the request allows a malicious attacker to submit a request on behalf of the user.</value>
```

Comment:
>Handling a POST, PUT and DELETE request without verifying that the request allows a malicious attacker to submit a request on behalf of the user. [](start = 11, length = 145)

Handling a POST, PUT, PATCH, or DELETE request without validating an antiforgery token may be vulnerable to cross-site request forgery attacks.  A cross-site request forgery attack can send malicious requests from an authenticated user to your ASP.NET Core MVC controller.